plugins/by-name/none-ls/prettierd.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, config, ... }:
+let
+  cfg = config.plugins.none-ls.sources.formatting.prettierd;
+  ts-ls-cfg = config.plugins.lsp.servers.ts_ls;
+in
+{
+  options.plugins.none-ls.sources.formatting.prettierd = {
+    disableTsServerFormatter = lib.mkOption {
+      type = with lib.types; nullOr bool;
```

Comment:
Why use `nullOr` here? Isn't it clearer to end-users what the default behaviour is if we just have a plain `bool` type?

Reply:
`nullOr bool` allows silencing the warning about `ts_ls` formatting conflict.

Reply:
Ah, I see. I've addressed this at https://github.com/nix-community/nixvim/pull/2488/files#r1838093274